frontend/src/components/FacilityBooking/HourBlocks.tsx

```diff
@@ -2,6 +2,13 @@ import React from 'react'
 
 import { MainTimeContainer, TimeContainer } from './BlockStyles.styled'
 
+/**
+ * @returns A column of each hour from 0000 to 2400, to be displayed next to DailyContainer containing time blocks
+ * @example
+ * ```
+ * <HourBlocks/>
+ * ```
```

Comment:
perhaps show some context? haha
```suggestion
 * @example
 * ```
 * // HourBlocks placed beside DailyContainer
 * <HourBlocks/>
 * <DailyContainer>...</DailyContainer>
 * ```
```